dashboard/app/models/foorm/form.rb

```diff
@@ -142,7 +136,22 @@ def readable_questions
     questions
   end
 
-  def parsed_questions
-    Pd::Foorm::FoormParser.parse_forms([self])
+  # Takes the questions in the survey (readable_questions)
+  # and adds in entries for metadata (eg, workshop_subject)
+  # that appears in the form submission, but not the form itself.
+  def merge_form_questions_and_config_variables(submission)
+    headers = readable_questions
+
+    # Get config variables by finding keys in the first form submission
+    # that aren't in the form itself.
+    # The header is the same as the key for these items (eg, workshop_subject).
+    config_variable_question_ids = submission.keys.reject do |question_id|
```

Comment:
we will run into an issue with picking the first submission if the first submission happens to be a facilitator survey (unlikely but still possible). Facilitator submissions don't contain metadata

Reply:
grr -- how can I filter out facilitator submissions?

Reply:
the associated record in `pd_workshop_survey_foorm_submissions` will have a non-null `facilitator_id`